Assets/Scripts/Movement/Cut.cs

```diff
@@ -20,12 +20,25 @@ public class Cut : MovementBase
 
         private void OnCutTriggerEnter(Collider other)
         {
+            // if (_objectToCut != null)
```

Comment:
Add this back, or add a queue of highlights.

![image](https://github.com/user-attachments/assets/035ec96b-0346-4430-b09b-b3fc8fb1a639)